fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckBottomSheet.kt

```diff
@@ -47,26 +47,14 @@ fun ReviewQualityCheckBottomSheet(
                     onPrimaryButtonClick = {
                         store.dispatch(ReviewQualityCheckAction.OptIn)
                     },
-                    onLearnMoreClick = { url ->
```

Comment:
I can't comment on the line but I think we can remove `LongMethod` suppression now